source/grid/manifold.cc

```diff
@@ -44,19 +44,34 @@ project_to_manifold (const std::vector<Point<spacedim> > &,
   return Point<spacedim>();
 }
 
-
+template <int dim, int spacedim>
+Point<spacedim>
+Manifold<dim, spacedim>::
+get_new_point (const Point<spacedim> &P1, const Point<spacedim> &P2, double w) const
+{
+  return w*P1 + (1-w)*P2;
```

Comment:
The default implementation should be 
~~~
std::vector<Point<spacedim> > vertices;
vertices.push_back(P1); vertices.push_back(P2);
project_to_manifold(vertices, w*P1 + (1-w)*P2);
~~~